src/boot/bootctl.c

```diff
@@ -112,6 +113,20 @@ static int acquire_xbootldr(bool unprivileged_mode, sd_id128_t *ret_uuid) {
         return 1;
 }
 
+static void settle_make_machine_id_directory(void) {
+        struct statfs stf;
+
+        if (arg_make_machine_id_directory != -1)
```

Comment:
doesn't matter, but when comparing these tri-states I'd always stick to comparing < 0, == 0, > 0, i.e. accept any value as long as it's relationship towards 0 is clear. i.e. change this to `if (arg_m_m_i_d >= 0)` maybe

Reply:
done